src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -308,19 +311,70 @@ public void Initialize()
         internal static string SanitizeHostJson(JObject hostJsonObject)
         {
             JObject sanitizedObject = new JObject();
-
             foreach (var propName in WellKnownHostJsonProperties)
             {
                 var propValue = hostJsonObject[propName];
                 if (propValue != null)
                 {
-                    sanitizedObject[propName] = propValue;
+                    sanitizedObject[propName] = Sanitize(propValue);
```

Comment:
There's existing tests here https://github.com/Azure/azure-functions-host/blob/e926662b1b1ae3c6479fd71b8419313272e2f062/test/WebJobs.Script.Tests/ScriptHostTests.cs#L1228. Can we update it to test this?